src/targets/github.ts

```diff
@@ -422,6 +334,24 @@ export class GithubTarget extends BaseTarget {
     }
   }
 
+  /**
+   * Publishes the release draft.
+   *
+   * @param release_id Release object ID
+   */
+  public async publishReleaseDraft(release_id: number) {
+    if (isDryRun()) {
+      this.logger.info(`[dry-run] Not publishing the release draft`);
+    } else {
+      await this.github.repos.updateRelease({
```

Comment:
Is having some protection of this method (like wrapping it in a `try..catch`) overengineering it? [Docs](https://octokit.github.io/rest.js/v18#repos-update-release) don't failure use cases when (e.g. release doesn't exist, an issue with GitHub...), but other methods [throw exceptions](https://github.com/getsentry/craft/pull/328/files#diff-5c75e39457a6946369f245a4d5672bd24e7bb641f645f1ae5412fe4f368cf145R437-R443).

Reply:
If the release doesn't exist we want to bail and find out about it, normal error handling seems fine.

If it's a transient network error I guess we would want to retry, but I'd rather not solve that as a one-off here, we shoudl address that across the board if we want.